blog/admin.py

```diff
@@ -1,7 +1,58 @@
+from blog.models import Blog, Comment, UserVote, Profile
+
 from django.contrib import admin
+from django.contrib.auth.admin import UserAdmin
+from django.contrib.auth.models import User
+
+
+class ProfileInline(admin.StackedInline):
+    model = Profile
+    can_delete = False
+    verbose_name_plural = 'Profile'
+    fk_name = 'user'
+
+
+class CustomUserAdmin(UserAdmin):
+    inlines = (ProfileInline, )
+    list_display = (
+        'get_profile_id', 'id', 'username', 'first_name', 'last_name', 'get_date_of_birth', 'email', 'get_gender',
```

Comment:
no need to display the `id`, `get_profile_id`

Reply:
It's been removed.